src/pages/SubmissionDetails.js

```diff
@@ -77,48 +148,30 @@ export const SubmissionDetails = ({ submissionId }) => {
               {
                 size: { basis: 350 },
                 field: 'record_id',
-                headerRenderer: () => h(Sortable, { sort, field: 'record_id', onSort: setSort }, ['Record Entry']),
+                headerRenderer: () => h(Sortable, { sort, field: 'record_id', onSort: setSort }, ['Workflow name']),
```

Comment:
As we discussed with Sara, this should probably be "<Whatever the record type is> ID" (eg "Sample ID" for samples, "Participant ID" for participants... "FOO ID" for "FOO"s...